FLEDGE.md

```diff
@@ -1136,20 +1146,36 @@ Note that the key fields are used by the browser both to verify the signature, a
 
 The browser ensures, using TLS, the authenticity and integrity of information provided to the auction through calls made directly to an ad tech's servers. This guarantee is not provided for data passed in `runAdAuction()`. To account for this, additional bids use the same HTTP response header interception mechanism that's already in use for the [Bidding & Auction response blob](FLEDGE_browser_bidding_and_auction_API.md#step-3-get-response-blobs-to-browser) and `directFromSellerSignals`.
 
-To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` fetch flag.
+ `Ad-Auction-Signals` response header 
```

Comment:
What is this line?  Is this a sub-section header? if so, can we add a ###### prefix
Actually this doesn't look like a header as I don't see further reference to Ad-Auction-Signals below.

Reply:
Good catch. This was a copy/paste error. Removed. Thanks!